cmd/mattermost/commands/export.go

```diff
@@ -230,7 +230,7 @@ func bulkExportCmdF(command *cobra.Command, args []string) error {
 	var opts model.BulkExportOpts
 	opts.IncludeAttachments = attachments
 	opts.CreateArchive = archive
-	if err := a.BulkExport(request.EmptyContext(a.Log()), fileWriter, filepath.Dir(outPath), opts); err != nil {
+	if err := a.BulkExport(request.EmptyContext(a.Log()), fileWriter, nil, filepath.Dir(outPath), opts); err != nil {
```

Comment:
Explore how to run this command and fix this.

Reply:
Ok..